shortcodes/pmpro_account.php

```diff
@@ -4,7 +4,7 @@
 */
 function pmpro_shortcode_account($atts, $content=null, $code="")
 {
-	global $wpdb, $pmpro_msg, $pmpro_msgt, $pmpro_levels, $current_user, $levels;
+	global $wpdb, $pmpro_msg, $pmpro_msgt, $current_user, $levels;

```

Comment:
I don't see $levels used anywhere in this. Perhaps that global isn't needed anymore either? Just asking - could be a separate PR.

Reply:
Going to remove $levels but also I don't think we need pmpro_msg or pmpro_msgt. Removing both - we will see if we find that these are needed. This code wasn't touched since 8+ years ago. I think it isn't used.